### PR TITLE
<TagList/> - use in Tag cheatsheet

### PR DESCRIPTION
**Pull request by gendeld**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5813/

File: stories/Introduction/Cheatsheet/componentsFamilies/OtherFamily.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ import {
   BadgeSelect,
   CounterBadge,
   Tag,
```
Comment (TalZa1810) [inline stories/Introduction/Cheatsheet/componentsFamilies/OtherFamily.js:28]: This is not longer used. Can be removed.

File: stories/Introduction/Cheatsheet/componentsFamilies/OtherFamily.js [hunk 2]
```diff
@@ -308,7 +312,7 @@ const TagsExample = () => {
 
   return (
     <SingleComponentSideBySide {...singleComponentProps}>
-      <Layout cols={6}>
+      <Layout cols={12}>
```
Comment (TalZa1810) [inline stories/Introduction/Cheatsheet/componentsFamilies/OtherFamily.js:315]: The default value is 12 so you can simply remove this.

File: stories/Introduction/Cheatsheet/componentsFamilies/OtherFamily.js (hunk 1 above)
Comment (gendeld) [reply stories/Introduction/Cheatsheet/componentsFamilies/OtherFamily.js:28]: It's used in line 291

**Pull request closed without merging**